pkgs/development/python-modules/pprintpp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchpatch, buildPythonPackage, fetchPypi, python, nose, parameterized }:
+
+buildPythonPackage rec {
+  pname = "pprintpp";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00v4pkyiqc0y9qjnp3br58a4k5zwqdrjjxbcsv39vx67w84630pa";
+  };
+
+  patches =
+    let
+      # https://github.com/wolever/pprintpp/pull/21
+      nose-parameterized-patch = fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/wolever/pprintpp/pull/21.patch";
+        sha256 = "0k7lfbyqi3fi4zcjnv9rm4kc22ip9bq9jvc50h3dbq1r5w428y92";
+      };
+    in [ nose-parameterized-patch ];
```

Comment:
Don't use `let`. You might have to use parantheses.

Reply:
PR URLs are stable once merged. Use https://github.com/wolever/pprintpp/commit/873217674cc824b4c1cfdad4867c560c60e8d806.patch instead.

Reply:
What about https://github.com/wolever/pprintpp/pull/21.patch?

Reply:
That might change if you or upstream add commits or force-push.

Reply:
But won't it be what we want it to be? And if I force push won't that commit maybe go away?

Reply:
Oh but the `sha256` might change and that will break it.